src/listen/websocket.rs

```diff
@@ -142,10 +144,110 @@ impl Stream for FileChunker {
 }
 
 impl<'a> StreamRequestBuilder<'a> {
-    pub fn keep_alive(mut self) -> Self {
-        self.keep_alive = Some(true);
+    /// Return the options in urlencoded format. If serialization would
+    /// fail, this will also return an error.
+    ///
+    /// This is intended primarily to help with debugging API requests.
+    ///
+    /// ```
+    /// use deepgram::{
+    ///     Deepgram,
+    ///     DeepgramError,
+    ///     common::options::{
+    ///         DetectLanguage,
+    ///         Encoding,
+    ///         Model,
+    ///         Options,
+    ///     },
+    /// };
+    /// # let mut need_token = std::env::var("DEEPGRAM_API_TOKEN").is_err();
+    /// # if need_token {
+    /// #     std::env::set_var("DEEPGRAM_API_TOKEN", "abc")
+    /// # }
```

Comment:
I don't love this pattern.  The `#`s ensure it doesn't show up in the docs, but it's still kind of ugly.

Reply:
What is the intent of those lines?

Reply:
I had a hard time fixing these imports in the examples in VSCode when they would break the build since there is no auto complete for them.